config/products.ts

```diff
@@ -29,19 +29,19 @@ const productSwitcherSlugs = [
 ]
 
 /**
- * Generates an array of Product objects from
+ * Generates a 2D array of Product objects from `productSwitcherSlugs`.
  */
-const products: ProductGroup[] = productSwitcherSlugs.map((slugGroup) => {
-  const productGroup = slugGroup.map((slug: ProductSlug) => {
-    const name = productSlugsToNames[slug]
-    const url = `/${slug}`
-    return {
-      name,
-      slug,
-      url,
-    }
-  })
-  return productGroup
-})
+const products: ProductGroup[] = productSwitcherSlugs.map(
+  (slugGroup: ProductSlug[]) => {
+    const productGroup = slugGroup.map((slug: ProductSlug) => {
+      const name = productSlugsToNames[slug]
+      return {
+        name,
+        slug,
+      }
+    })
+    return productGroup
+  }
+)
```

Comment:
Couple of changes here:

- The outer `map` call had a typedef warning on `productSlug`
- The `url` property became irrelevant with some other changes, so I removed it
- The comment above this was incomplete, so I updated it
- I felt it was beneficial to add a type of `productSwitcherSlugs` to remove ambiguity